hack/test

```diff
@@ -63,7 +63,7 @@ fi
 if [ "$TEST_INTEGRATION" == 1 ]; then
   cid=$(docker create --rm -v /tmp $coverageVol --volumes-from=$cacheVolume -e TEST_DOCKERD -e SKIP_INTEGRATION_TESTS ${BUILDKIT_INTEGRATION_SNAPSHOTTER:+"-eBUILDKIT_INTEGRATION_SNAPSHOTTER"} -e BUILDKIT_REGISTRY_MIRROR_DIR=/root/.cache/registry --privileged $iid go test $coverageFlags ${TESTFLAGS:--v} ${TESTPKGS:-./...})
   if [ "$TEST_DOCKERD" = "1" ]; then
-    docker cp "$TEST_DOCKERD_BINARY" $cid:/usr/bin/
+    docker cp "$TEST_DOCKERD_BINARY" $cid:/usr/bin/dockerd
```

Comment:
necessary for https://github.com/moby/moby/pull/43239/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR83